src/trace/descriptor.ts

```diff
@@ -36,7 +36,17 @@ export function augmentLogs(logs: StructLog[], constants: object): { logs: Log[]
     const nextStack = nextLog ? nextLog.stack : undefined;
 
     let { extended, info: nInfo } = describeOperation(log, nextStack || [], info);
-    const nLog: Log = new Log(log, extended, info.lastLog);
+    let nLog: Log = new Log(log, extended, info.lastLog);
```

Comment:
FWIW, I'm always so torn on const. I think it's dumb, but it does technically do the same thing without allowing rebinds. But I think `let` feels nicer, regardless.